config.inc.php

```diff
@@ -86,6 +86,8 @@ function language_hook($PALANG, $language) {
 $CONF['database_user'] = 'postfix';
 $CONF['database_password'] = 'postfixadmin';
 $CONF['database_name'] = 'postfix';
+// Default mysql port for mysqli connection
+$CONF['database_port'] = '3308';
```

Comment:
URI host:port not work for mysqli. :(
Mysqli constructor : http://php.net/manual/fr/mysqli.construct.php